superset-frontend/src/dashboard/stylesheets/dashboard.less

```diff
@@ -19,6 +19,7 @@
 /* header has mysterious extra margin */
 header.top {
   margin-bottom: 2px;
+  z-index: 10;
```

Comment:
why is this necessary?

Reply:
The main menu items on the right were not clickable because in edit mode the right panel was covering them (they were visible but unclickable because of invisible part of that panel covering them)